checker/tests/nullness/Issue2587.java

```diff
@@ -0,0 +1,35 @@
+import java.util.HashMap;
+import java.util.Map;
+import org.checkerframework.checker.nullness.qual.KeyFor;
+
+public abstract class Issue2587 {
+    public enum EnumType {
+        // :: error: (expression.unparsable.type.invalid) :: error: (assignment.type.incompatible)
```

Comment:
I think that a good idea would be to `@SuppressWarnings("assignment.type.incompatible")` for the entire class. These errors only distract from the actual issue.

Reply:
I agree, just make sure you document why you added the `@SuppressWarnings`.